src/main/java/no/nav/RateLimiterFilter.java

```diff
@@ -0,0 +1,30 @@
+package no.nav;
+
+import static java.util.concurrent.TimeUnit.SECONDS;
+import static javax.ws.rs.core.Response.status;
+import static javax.ws.rs.core.Response.Status.TOO_MANY_REQUESTS;
+
+import java.io.IOException;
+
+import javax.ws.rs.container.ContainerRequestContext;
+import javax.ws.rs.container.ContainerRequestFilter;
+import javax.ws.rs.ext.Provider;
+
+import com.google.common.util.concurrent.RateLimiter;
+
+@Provider
```

Comment:
Jeg vurdertre å legge inn `@PreMatching`-annotasjonen her, slik at filteret kjøres også dersom det ikke finnes noen matchende ressurs for url-en, men vet ikke om det har noen hensikt. Dropper det iallfall inntil videre.